s/frontend/components/banned-users/index.ts

```diff
@@ -46,7 +46,7 @@ export class NamakaBans extends LitElement {
 									<td>
 										<div class="user">
 											<div class="box">
-												<img src="/assets/profile-img.png" alt="Profile Image">
+												<img src="https://source.unsplash.com/random?sig=1234" alt="Profile Image">
```

Comment:
these images are too big. is there a way we can request images sized at like 200x200 from unsplash?